client/engine/store/mockstore.go

```diff
@@ -13,35 +14,75 @@ import (
 )
 
 type MockStore struct {
-	objectives map[protocols.ObjectiveId][]byte
-	channels   map[types.Destination][]byte
+	objectives bytesSyncMap
+	channels   bytesSyncMap
 
 	key     []byte        // the signing key of the store's engine
 	address types.Address // the (Ethereum) address associated to the signing key
 }
 
+// bytesSyncMap wraps sync.Map in order to provide type safety
+type bytesSyncMap struct {
+	m sync.Map
+}
+
+// Load returns the value stored in the map for a key, or nil if no
+// value is present.
+// The ok result indicates whether value was found in the map.
```

Comment:
Nit: does consuming understand the "ok result"? Or do they only understand that the second return value is a boolean?

Reply:
I think the `sync` package (where this godoc is copied from) actually names the return parameter, so the comment makes more sense there (I think?)

Reply:
fixed in d4deb77

note also that these are public functions on a private struct - the given OK return does not become part of mockstore's API